packages/frontend/electron/renderer/shell/index.tsx

```diff
@@ -2,6 +2,7 @@ import 'setimmediate';
 import '@affine/component/theme/global.css';
 import '@affine/component/theme/theme.css';
 import '@affine/core/bootstrap/preload';
+import '../global.css';
```

Comment:
is it safe ?

Reply:
on production global.css is loaded for both shell/renderer already.
tried both dev/prod locally and it works fine.